src/Files.App/Actions/Content/Archives/DecompressArchiveToChildFolderAction.cs

```diff
@@ -1,24 +1,19 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using CommunityToolkit.Mvvm.DependencyInjection;
 using Files.App.Contexts;
-using Files.App.Extensions;
-using Files.App.Helpers;
-using System.ComponentModel;
-using System.IO;
```

Comment:
This is needed

Reply:
Those were grayed out.

Reply:
My bad, the only one needed was `System.IO` (which you already added back). I selected too many rows